api/MutationObserver.json

```diff
@@ -0,0 +1,473 @@
+{
+  "api": {
+    "MutationObserver": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/MutationObserver",
+        "support": {
+          "webview_android": {
+            "version_added": null
+          },
```

Comment:
This should be:
```json
"webview_android": [
  {
    "version_added": "26"
  },
  {
    "version_added": "18",
    "version_removed": "26",
    "prefix": "Webkit"
  }
]
```
And moved to the end of the `support` block (see #398 and #1882).